session/session.go

```diff
@@ -2848,6 +2894,10 @@ func BootstrapSession(store kv.Storage) (*domain.Domain, error) {
 			return nil, err
 		}
 	}
+	err := InitMetaTable(store)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Why not run the initialization it in `doDDLWorks()`?

Reply:
because `doDDLWorks` only run if it is a new cluster.